config/linchpin.conf

```diff
@@ -97,8 +97,8 @@ enable_uhash = True
 # in older versions of linchpin (<v1.0.4), a resources folder exists, which
 # dumped the data that is now stored in the RunDB. To disable the resources
 # output, set the value to False.
-#generate_resource = True
-generate_resource = False
+#generate_resources = True
+generate_resources = False
```

Comment:
If the backwards-compatible resources output is off by default, then a lot of existing linchpin users would need to update a large number of CI jobs to turn this back on.  Can we set this to **True** and generate the resources files by default so existing CI jobs that invoke linchpin can continue to work without modification?

Reply:
Hi @robled. What you may have missed was a slight adjustment made here. PR #460 explains the adjustment.  Essentially, there is now a set of defaults, managed by linchpin.constants. In that [file](https://github.com/CentOS-PaaS-SIG/linchpin/blob/develop/linchpin/linchpin.constants#L55), you will see that generate_resources is set to **False** as desired. The config/linchpin.conf no longer controls the defaults in linchpin. The linchpin.conf you referenced here is provided as an example and doesn't change the settings. 

The reason for this change is so that backward compatibility can be set as the default without having to do anything. Overriding settings, as you have been doing with the linchpin.conf in the workspace should continue to work as normal. Please test this and verify that indeed the resources are being created as you expect. If you find any errors, please let me know.